src/main/java/javax/jmdns/impl/constants/DNSConstants.java

```diff
@@ -15,9 +15,9 @@
     // changed to final class - jeffs
     public static final String MDNS_GROUP                     = "224.0.0.251";
     public static final String MDNS_GROUP_IPV6                = "FF02::FB";
-    public static final int    MDNS_PORT                      = Integer.parseInt(System.getProperty("net.mdns.port", "5353"));
+    public static final int    MDNS_PORT                      = Integer.getInteger("net.mdns.port", 5353);
     public static final int    DNS_PORT                       = 53;
-    public static final int    DNS_TTL                        = 60 * 60;                                                      // default one hour TTL
+    public static final int    DNS_TTL                        = Integer.getInteger("net.dns.ttl", 60 * 60);                   // default one hour TTL
     // public static final int DNS_TTL = 120 * 60; // two hour TTL (draft-cheshire-dnsext-multicastdns.txt ch 13)
```

Comment:
Thanks for this PR, looks good to me (and sorry for the late response!).

Just one question: This comment correctly states that the TTL should be 2 hours according to the mDNS spec. Do you know why it is nonetheless set to 1 hour as a default in the code?